test.ts

```diff
@@ -118,58 +264,291 @@ describe('supported value types', () => {
 
 describe('supported flag types', () => {
   it('flag with two dashes', () => {
-
+    const [options] = parse(
+      ['--yesterday'],
+      {
+        'yesterday': {
+          alias: 'y',
+          desc: 'oh I believe in yesterday.',
+          type: 'boolean'
+        }
+      }
+    );
+    expect(options).toEqual({
+      yesterday: true
+    });
   });
 
   it('flag with single dash', () => {
-
+    const [options] = parse(
+      ['-q', 'three'],
+      {
+        'quiet': {
+          alias: 'q',
+          desc: 'the quiet one.',
+          type: 'string'
+        }
+      }
+    );
+    expect(options).toEqual({
+      quiet: 'three'
+    });
   });
 
+  // TODO: multiple alias in single dash ?
```

Comment:
"-hvs" means ["--help", "--version", "--silent"] if you define these rules with the alias names